basic-examples/serial-data_passing/README.md

```diff
@@ -0,0 +1,62 @@
+# Implicit Serial Workflow
+
+## Workflow Description
+
+By default, multiple steps in a workflow will be run in parallel if there is no data
+passing relationship between the steps. When the input of a step relies on the output
+of another step, an implicit serial relationshp is created.
+
+This workflow runs a metadata collection plugin step and then an example plugin (in this
+case using the template plugin) that takes its `name` input from the `fqdn` output
+returned by the metadata plugin. All steps are run via the default deployer (defined in
+`config.yaml` as podman) and their success outputs are reported.
```

Comment:
Do we really want to say "default", when it's being explicitly specified in the config file?

Do we really want to say which deployer it is, here, when it is defined in the config file?  (I.e., if we change our mind again, we now have _two_ places which have to be modified....)

Are we sure we even want to use a config file?  Wouldn't it be nicer to use the default (built-in) one?

And, we're missing the comma before "and".